packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -227,6 +262,85 @@ class MessageList extends Widget<Properties> {
     });
   }
 
+  _getContextMenuButtons(message: Message): ContextMenuItem[] {
+    const {
+      allowUpdating,
+      allowDeleting,
+      onMessageEditingStart,
+      onMessageDeleting,
+    } = this.option();
+
+    const editText = messageLocalization.format('dxChat-editingEditMessage');
```

Comment:
it's bette to place it near the place of usage inside of `if`. Now we get this data even when it's no needed